appveyor.yml

```diff
@@ -28,7 +29,7 @@ build_script:
 after_build:
   - cd ..
   - cd win32port
-  - 'C:\Program Files\NSIS\makensis.exe -DVERSION=%APPVEYOR_BUILD_VERSION% libwebsockets.nsi'
+  - makensis -DVERSION=%APPVEYOR_BUILD_VERSION% libwebsockets.nsi
```

Comment:
Only build this when `LWS_METHOD` is `default` (not sure how to check this in appveyor)

`if LWS_METHOD == default then makensis......`